packages/developer-console-spa/src/utils/gqlClient.ts

```diff
@@ -4,6 +4,13 @@ type GQLRequestProps = {
   operationName?: any;
 };
 
+const handleFailure = ( res: any ) => {
+  if ( !res.ok ) {
+  console.debug( '[GQLClient]:' + res.statusText );
+  }
+  return res;
+};
+
```

Comment:
Why is this separated in a different function? It is totally fine in the `.then`, as this is not really useful for anything else and hence cannot be reused.